java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java

```diff
@@ -188,6 +243,9 @@ private String getNewIssueLink(String phoneNumber, String defaultCountry)
         "Supporting evidence (for example, national numbering plan, announcement from mobile "
         + "carrier, news article): **IMPORTANT - anything posted here is made public. "
         + "Read the guidelines first!** \n\n");
+    issueTemplate.append("[link to demo]("
```

Comment:
Do we even want this? We just want the page on which results are shown to be bookmarkable. No need to add any more UI elements or anything.